src/psyclone/transformations.py

```diff
@@ -1285,9 +1298,6 @@ class to convert the Loop represented by :py:obj:`node` into a
                 "Error in DynamoColour transformation. Loops iterating over "
                 "a discontinuous function space are not currently supported.")
 
-        # Check that we don't have an inter-grid kernel
-        check_intergrid(node)
-
```

Comment:
Please document the associated method arguments ( apply() )

Reply:
Done.

Reply:
Great, thanks.